test/create-run-publish-app.sh

```diff
@@ -33,5 +25,8 @@ if [ "rel-1.0.0-preview2.1" == "${2}" ]; then
     dotnet run
     dotnet publish -o publish/self-contained
 else
-    dotnet publish3 -r debian.8-x64 -o publish/self-contained
+    sed -i '/<PropertyGroup>/a \    <RuntimeIdentifiers>debian.8-x64<\/RuntimeIdentifiers>' ./${1}.csproj
```

Comment:
At minimum, adding a comment mentioning that $1 should be at the root level would be helpful.